### PR TITLE
refactor(gazelle_cli): Improvement of test cases & method of handing strings

### PR DESCRIPTION
**Pull request by raman04-byte**
I have introduced a new way of handling strings and added another folder for such a case if there is something common in the test cases we can teach in the common folder. 

The app_strings is not completed, I need your review on whether should I continue this approach or change the approach do let me know.

### CODE REVIEW

**Comment by filippomenchini** [comment]
LGTM, I think you can continue this way!

**Comment by raman04-byte** [comment]
There are several other modification I have to do into this PR so don't merge yet.

**Comment by filippomenchini** [comment]
I like it!
The only thing I would change is the `resources.dart` file. I think it should be named `constants.dart`, other than that, looks good to me!

**Comment by raman04-byte** [comment]
Changed the filename from `resources.dart` to `constants.dart`

**Comment by filippomenchini** [comment]
LGTM!

Notice for the next PRs:
Please use [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) to commit
your code. This is important to keep the repository clean and in order.

File: packages/gazelle_cli/lib/commands/delete/delete_project.dart [hunk 1]
```diff
@@ -6,7 +6,7 @@ class DeletingProjectError {
   final String message;
 
   /// Creates a [DeletingProjectError].
-  const DeletingProjectError() : message = "Failed to delete project!";
+  const DeletingProjectError() : message = "No Gazelle project found.";
```
Comment (filippomenchini) [inline packages/gazelle_cli/lib/commands/delete/delete_project.dart:9]: I would change this class name to `DeleteProjectError`.

**Review by filippomenchini — APPROVED** [review]
LGTM!

File: packages/gazelle_cli/lib/commands/delete/delete_project.dart (hunk 1 above)
Comment (raman04-byte) [reply packages/gazelle_cli/lib/commands/delete/delete_project.dart:9]: DeletingProjectError is changed to DeleteProjectError

**Pull request merged**